runtime/doc/autocmd.txt

```diff
@@ -78,6 +78,9 @@ exception is that "<sfile>" is expanded when the autocmd is defined.  Example:
 	:au BufNewFile,BufRead *.html so <sfile>:h/html.vim
```

Comment:
@zeertzjq I left the mention of `<sfile>` here for now. All other changes should be correct now.